### DISCUSSION TITLE
MCprep 3.5 RC-2

### DISCUSSION THREAD

**Discussion by StandingPadAnimations** [Moo-Ack-Productions/MCprep#475] [Release candidates]
Release candidate 2 for MCprep 3.5 is now out:
https://github.com/Moo-Ack-Productions/MCprep/releases/download/untagged-6406ec0d07fc3ebd7818/MCprep_addon.zip

This release adds the addon updater ignore filter for release candidates and removes all 2.7x related code.

Also it's my birthday today :D

**Comment by TheDuckCow** [comment]
Happy birthday @StandingPadAnimations! Thanks for making another build
